gateway/service/src/main/resources/gateway.conf

```diff
@@ -63,7 +63,9 @@ ditto {
     # The factory for the gateway authentication directive
     gateway-authentication-directive-factory = org.eclipse.ditto.gateway.service.endpoints.directives.auth.DittoGatewayAuthenticationDirectiveFactory
     # Creator of props of HTTP request actors. Must implement HttpRequestActorPropsFactory.
-    http-request-actor-props-factory = org.eclipse.ditto.gateway.service.endpoints.actors.DefaultHttpRequestActorPropsFactory
+    http-request-actor-props-factory = org.eclipse.ditto.gateway.service.endpoints.actors.DefaultHttpRequestActorPropsFactory    # Creator of props of HTTP request actors. Must implement HttpRequestActorPropsFactory.
+    # Creator of props of CheckPermissions actors. Must implement CheckPermissionsActorPropsFactory.
+    check-permissions-actor-props-factory = org.eclipse.ditto.gateway.service.endpoints.actors.DefaultCheckPermissionsActorPropsFactory
```

Comment:
I think that can be removed - we don't use it as an extension as far as I could see.